itou/employee_record/common_management.py

```diff
@@ -166,29 +166,33 @@ def preflight(self, object_class):
         )
 
         if not new_objects:
-            self.stdout.write("No object to check. Exiting preflight.")
+            self.logger.info("No object to check. Exiting preflight.")
             return
 
-        self.stdout.write(
-            f"Found {len(new_objects)} object(s) to check, split in chunks of "
-            f"{EmployeeRecordBatch.MAX_EMPLOYEE_RECORDS} objects."
+        self.logger.info(
+            "Found %d object(s) to check, split in chunks of %d objects.",
+            len(new_objects),
+            EmployeeRecordBatch.MAX_EMPLOYEE_RECORDS,
         )
 
         errors = False
         for idx, elements in enumerate(batched(new_objects, EmployeeRecordBatch.MAX_EMPLOYEE_RECORDS), 1):
             # A batch + serializer must be created with notifications for correct serialization
             batch = EmployeeRecordBatch(elements)
 
-            self.stdout.write(f"Checking file #{idx} (chunk of {len(elements)} objects)")
+            self.logger.info("Checking file #%d (chunk of %d objects)", idx, len(elements))
 
             for obj in batch.elements:
                 try:
                     object_serializer(obj).data  # Invoke DRF serialization
                 except Exception as ex:
-                    self.stdout.write(f"ERROR: serialization of {obj} failed!")
-                    self.stdout.write("".join(f"> {line}" for line in str(ex).splitlines(keepends=True)))
+                    self.logger.exception(
+                        "ERROR: serialization of %s failed!\n%s",
```

Comment:
L'avantage des loggers c'est que le niveau est plus clair :)
```suggestion
                        "Serialization of %s failed!\n%s",
```